canonical/ubuntu.com
# Update /pro/subscribe page

## Done
1. Disable Ubuntu Pro in the step 4 "What security coverage do you need?" when server 14.04 is selected
2. Add public cloud instances and update copy doc according to this [doc](https://docs.google.com/document/d/1UqL8YJH9ywvOXnrt3syY0U8fiOakfAGWmLS9uerwAWk/edit)
2. Add "Who will be using this subscription?" step according to this [figma](https://www.figma.com/file/FEK76z86SxEzzRUzSOcpKe/Pro-Shop-update---Personal-vs-Enterprise-subscription?type=design&node-id=20-8&mode=design)

## QA
- go to /pro/subscribe
1. 
- Select "myself"  in the Step 1 "Who will be using this subscription?" and check all other steps are not visible and in the summary it says "ubuntu pro", quantity 5, total free, button says register. 
- Select "my organisation" and check all other steps are working as it was before
2. 
- Select "my organisation"  in the Step 1 
- Select `Physical servers with unlimited VMs` in the Step 2
- Select `14.04 LTS` in the step 4
- See Ubuntu Pro is disabled in the step 5 "What security coverage do you need?" 
3.
- Select "my organisation"  in the Step 1 
- Select `Public cloud instances` in the Step 2
- See public cloud instances buttons are updated properly according to this [doc]

## Issue / Card

Fixes #https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-4588
and https://warthogs.atlassian.net/browse/WD-4519
